config/locales/en/summary.yml

```diff
@@ -740,9 +740,6 @@ en:
       # END national_savings_certificates
 
       # BEGIN property, address and owners
-      house_type:
-        question: Type of property
-        absence_answer: *absence_none
```

Comment:
As the summary card displays the house type (e.g. Residential), this row is actually not required